src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -395,6 +395,8 @@ private function getToolbarButtons()
      * Create a new basic product
      * Then return to form action.
      *
+     * @AdminSecurity("is_granted('create', request.get('_legacy_controller'))")
```

Comment:
This is the product page V1. I am not sure at all :/ that the `_legacy_controller` property will be set correctly in that backoffice page. I think we should not put AdminSecurity on Page Product V1 controllers :/

Reply:
Indeed the `_legacy_controller` attribute is not set everywhere in the routing file, but it seems like they have been added in this PR so it's fine